CONTRIBUTING.md

```diff
@@ -151,7 +151,7 @@ source ~/.bash_profile  # MAC
 
 **Note:** the following steps guide you through the manual process of creating a development environment.
 For convenience, there is a script that achieves the same effect located in `ilastik-meta/ilastik/scripts`.
-You can create a development environment (given miniconda is installed) by invoking `./create-devenv.sh -c ilastik-forge -c conda-forge idev ../../` from within `ilastik-meta/ilastik/scripts`.
+You can create a development environment (given anaconda or miniconda is installed) by invoking `python ilastik/scripts/devenv.py -e idev create` from within `ilastik-meta`.
```

Comment:
shouldn't you indicate the version WITH linking to the existing ilastik-meta (ie specifying a `--location`)?

Reply:
By default, location is the current working directory, you can change it with `-l|--location`.

Reply:
then you should at least note that you have to do it from `ilastik-meta` (which is, of course implicitly given in your command).

Reply:
It is mentioned in the doc:
> ... from within `ilastik-meta`

I'll reword it anyway to make this read crystal clear.

Reply:
uh sorry - my bad :/